reference/platform-app-yaml.md

```diff
@@ -2,47 +2,50 @@
 
 ## Configure your Application
 
-Platform.sh supports multiple applications per project (for example, a RESTful
-web service and a front-end, or a main web site and a blog). But more often
-than not, projects are composed of a single application, in which case you can
-simply put a `.platform.app.yaml` at the root of your repository.
-
-This file controls the application and the way it will be built and deployed on Platform.sh.
+You control your application and the way it will be built and deployed on Platform.sh
+via a single configuration file `.platform.app.yaml` located at the root of your application 
+folder inside your Git repository.
 
 Here is an example of a `.platform.app.yaml` file for Drupal:
 
-```yaml
-name: front
-type: php:5.5
-build:
-    flavor: drupal
-relationships:
-    database: "mysql:mysql"
-    solr: "solr:solr"
-    redis: "redis:redis"
-web:
-    document_root: "/"
-    passthru: "/index.php"
-disk: 2048
-mounts:
-    "/public/sites/default/files": "shared:files/files"
-    "/tmp": "shared:files/tmp"
-    "/private": "shared:files/private"
-hooks:
-    # We run deploy hook after your application has been deployed and started.
-    deploy: |
-        cd public
-        drush -y updatedb
-crons:
-    drupal:
-        spec: "*/20 * * * *"
-        cmd: "cd public ; drush core-cron"
-```
+    name: front
+    type: php:5.5
+    build:
+        flavor: drupal
+    relationships:
+        database: "mysql:mysql"
+        solr: "solr:solr"
+        redis: "redis:redis"
+    web:
+        locations:
+            "/":
+                root: "public"
+                expires: -1
+                passthru: "/index.php"
+                index: 
+                    - index.php
+                allow: true
```

Comment:
@pjcdawkins I think it's fine as this for an example. We probably don't want to have too many options in an example.